cafebabel/static/js/script.js

```diff
@@ -113,3 +113,31 @@ Array.from(document.querySelectorAll('h1.edit input')).forEach(inputh1 => {
     () => inputh1.value || h1.classList.remove('active')
   )
 })
+
+/* Auto-expand textarea when typing http://jsfiddle.net/hmelenok/WM6Gq/ */
```

Comment:
This whole script being an external library, the functions it should be in an isolated file and then called here like you did on line 143.

Reply:
ok done. https://github.com/cafebabel/cafebabel.com/pull/220/commits/d149accdc65015f66500169fea80accaed71772f